bin/mail/2016-04-26-changeorg-petition/send-ab-test-real-mails

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```

Comment:
On another note, we might consider changing the naming schema in `split` to not use the suffix "test" in the generated files.